lang/solidity/Portfile

```diff
@@ -24,36 +23,20 @@ long_description    Solidity is a statically typed, contract-oriented, \
                     high-level language for implementing smart contracts \
                     on the Ethereum platform.
 homepage            https://soliditylang.org
-master_sites        https://github.com/ethereum/solidity/releases/download/v${github.version}/
 
-checksums           rmd160  df322fa402d7fc3d1c6643c0136a44251189ce28 \
-                    sha256  9bc195e1695f271b65326c73a12223377d52ae4b4fee10589c5e7bde6fa44194 \
-                    size    3239230
+checksums           rmd160  14a80ba6315744ec6892405a91b0ce22195cb78f \
+                    sha256  def2ab7c877fcd16c81a166cdc5b99bfabcee7e8d505afcce816e9b1e451c61a \
+                    size    3386993
 
-set port_libfmt     libfmt9
-
-depends_build-append \
-                    port:${port_libfmt} \
-                    port:range-v3
-depends_lib         port:jsoncpp
-
-cmake.module_path-append \
-                    ${prefix}/lib/${port_libfmt}/cmake
+depends_lib-append  port:boost
```

Comment:
please keep using the "boost" PortGroup; what's the reason you changed?

Reply:
When I used PortGroup to compile, Boost could not be linked correctly.  So I used `depends_lib` to link boost from MacPorts.